topmenubar/src/main/java/de/interseroh/tmb/client/TopMenuBarWebApp.java

```diff
@@ -94,12 +98,16 @@ private void createViews() {
 		injectApplauncherScript();
 		configureRootPanel();
 
-		String color = rootPanel.getElement().getAttribute(DATA_TMB_COLOR);
+		String color = rootPanel.getElement().getAttribute(DATA_TMB_BGCOLOR);
+		String text_color = rootPanel.getElement().getAttribute(DATA_TMB_TXTCOLOR);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00117)